bimsb/packages/bioinformatics-nonfree.scm

```diff
@@ -1882,3 +1883,95 @@ MACE is a bioinformatics tool dedicated to analyze ChIP-exo data.")
     ;; others say "free for all use", others say "GPL version 1.3" (?), yet
     ;; others say "Artistic license".
     (license license:gpl2+)))
+
+;; Although this program is released under the GPL it depends on
+;; ViennaRNA and bowtie1, which are non-free software.
+(define-public mirdeep2
+  (package
+    (name "mirdeep2")
+    (version "0.0.92")
+    (source (origin
+              (method git-fetch)
+              (uri (git-reference
+                   (url "https://github.com/rajewsky-lab/mirdeep2.git")
+                   (commit (string-append "v" version))))
+              (file-name (string-append name "-" version "-checkout"))
+              (sha256
+               (base32
+                "1gasx6vb38n4vbjbn5s7ch2rpflgf2crdd8mdp2xm571w2j01wmm"))))
+    (build-system gnu-build-system)
+    (arguments
+     `(#:tests? #f ;; no check target
+       #:phases
+       (modify-phases %standard-phases
+         (delete 'configure)
+         (replace 'build
+           (lambda* (#:key inputs outputs #:allow-other-keys)
+             (let ((out (assoc-ref outputs "out")))
+               ;; patch scripts checking for ../install_successful file
+               (for-each
+                 (lambda (script)
+                   (substitute* script
+                     (("\\$bn/../install_successful")
+                      (string-append out
+                                     "/share/mirdeep2/install_successful"))))
```

Comment:
This will actually hard-code `/gnu/store/...-mirdeep2-.../share/mirdeep2/...` instead of `/var/guix/profiles/.../guix-profile/share/mirdeep2/...` - not sure if that's the right way to do it as clearly `~/guix-profile/share/mirdeep2/...` points to the right place.